src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookTriggerImpl.java

```diff
@@ -129,5 +155,42 @@ private void registerWebhook(BitbucketSCMRepository repository) {
             BitbucketWebhook webhook = retryingWebhookHandler.register(repository);
             LOGGER.info("Webhook returned -" + webhook);
         }
+
+        private boolean isWebhookExists(Job<?, ?> project, BitbucketSCM input) {
+            try (ACLContext ctx = ACL.as(ACL.SYSTEM)) {
+                return jenkinsProvider
+                        .get().getAllItems(ParameterizedJobMixIn.ParameterizedJob.class)
+                        .stream()
+                        .filter(item -> !item.equals(project))
+                        .filter(this::isTriggerEnabled)
+                        .map(item -> asSCMTriggerItem(item))
+                        .filter(Objects::nonNull)
+                        .map(scmItem -> scmItem.getSCMs())
+                        .flatMap(Collection::stream)
+                        .filter(scm -> scm instanceof BitbucketSCM)
+                        .map(scm -> (BitbucketSCM) scm)
+                        .map(BitbucketSCM::getRepositories)
```

Comment:
I think this would read a little better if you combined the two lines:
`.map(scm -> ((BitbucketSCM) scm).getRepositories())`